### DISCUSSION TITLE
LND ignores custom datadir

### DISCUSSION THREAD

**Discussion by whatafuiamdoing** [lightningnetwork/lnd#8284] [Troubleshooting]
### **Issue**

**First case**

When running 2 nodes simultaneously, the second node returns an error: unable to open databases: unable to obtain database backends: error opening bolt DB: timeout

Created data directories for both nodes in `C:\Users\****\AppData\Local\Lnd\lnd.conf` & `C:\Users\****\AppData\Local\Lnd-2\lnd.conf,` and 2 directories for lnd node: 

AppData: 
     Local:
          lnd-windows-amd64-v0.1-beta:
              lnd.exe
              lncli.exe
          lnd-windows-amd64-v0.1-beta2:
              lnd.exe
              lncli.exe

Running 1 node `lnd.exe --rpclisten=127.0.0.1:10001 --listen=127.0.0.1:10011 --restlisten=localhost:8001` succesfully, but when i try to run second node `lnd.exe --rpclisten=127.0.0.1:10002 --listen=127.0.0.1:10012 --restlisten=localhost:8002 --datdir="C:\Users\*****\AppData\Local/Lnd-2"`, get an error: **unable to open databases: unable to obtain database backends: error opening bolt DB: timeout**.

Lnd ignores ---datadir flag and interact with default LND dir.

```
lnd.conf
[Bitcoin]
bitcoin.active=1
bitcoin.node=bitcoind
bitcoin.regtest=1
[bitcoind]
bitcoind.rpcuser=******
bitcoind.rpcpass=******
bitcoind.zmqpubrawblock=tcp://127.0.0.1:28332 
bitcoind.zmqpubrawtx=tcp://127.0.0.1:28333
[Application Options]
externalip=145.145.145.145
color=#FF9900
alias=NodeA
datadir=data
```

**Second case**

Creating 2 directories 
`
[Direcotry 1]
Lnd: 
  lnd.exe
  lncli.exe
  lnd.conf
[Directory 2]
Lnd-2: 
  lnd.exe
  lncli.exe
  lnd.conf
`
lnd.conf
`
debuglevel=debug
alias=NodeA
maxpendingchannels=10
color=#eeeeee
rpclisten=0.0.0.0:10100
restlisten=0.0.0.0:8090
listen=9736
externalip=127.0.0.1:9736
datadir=./
tlskeypath=./tls.key
adminmacaroonpath=./data/chain/regtest/admin.macaroon

bitcoin.active=1
bitcoin.regtest=1
bitcoin.node=bitcoind

bitcoind.rpcuser=******
bitcoind.rpcpass=******
bitcoind.zmqpubrawblock=tcp://127.0.0.1:28332
bitcoind.zmqpubrawtx=tcp://127.0.0.1:28333`

running with `lnd --configfile=./lnd.conf`

the same issue

**Comment by guggero** [comment]
See my answer here: https://github.com/lightningnetwork/lnd/issues/8285#issuecomment-1859706313
